StripeFinancialConnections/StripeFinancialConnections/Source/Native/ManualEntry/ManualEntryFormView.swift

```diff
@@ -82,15 +87,24 @@ final class ManualEntryFormView: UIView {
         return (routingNumberTextField.text, accountNumberTextField.text)
     }
 
-    init() {
+    init(isTestMode: Bool) {
         super.init(frame: .zero)
         let contentVerticalStackView = UIStackView(
             arrangedSubviews: [
                 textFieldStackView,
             ]
         )
+
+        if isTestMode {
+            let testModeBanner = TestModeAutofillBannerView(
+                context: .account,
+                didTapAutofill: applyTestModeValues
```

Comment:
On Android and Web, we also submit the input on `Use test code` clicks instead of just filling it in. Should we do the same on iOS?

Reply:
Ouu good catch, yeah we could do that!